lua/hbac/telescope/make_finder.lua

```diff
@@ -1,9 +1,11 @@
 local finders = require("telescope.finders")
 local results_opts = require("hbac.telescope.results_opts")
 
-local M = {}
+local M = {
+	default_selection_idx = 1,
+	finder_opts = {},
+}
```

Comment:
I was also storing the `finder_opts` in this table to be read on picker refreshes (after toggling pin states), so I set the default here as well.